def logError(msg):
    log(msg, 4)
